src/api-routes/books/{id}.js

```diff
@@ -22,4 +22,34 @@ const get = async (req, res) => {
   }
 };
 
-export { get };
+/**
+ * Update book by unique ID
+ *
+ * @type {RequestHandler}
+ */
+const patch = async (req, res) => {
+  try {
+    const { id } = req.params;
```

Comment:
Sorry, I might miss that when reviewed your openapi design. It's better to name your path parameter more descriptive something like here: https://github.com/osu-mist/onbase-docs-api/blob/v2/openapi.yaml#L134

With this change, you may want to update your openapi as well.

Reply:
like library/books/{book-id}?

Reply:
done.